birdcage_backend/app/static/openapi.yaml

```diff
@@ -187,7 +187,7 @@ paths:
               schema:
                 $ref: '#/components/schemas/Message'
 
-  /audio-files/{filename}:
+  /api/audio-files/{filename}:
```

Comment:
Fixing some inconsistencies I found, not every api endpoint had an `/api/` prefix which was super confusing to troubleshoot when you're messing with `API_SERVER_URL`

Reply:
Thanks! I had been meaning to do that.